pkg/env/prod.go

```diff
@@ -154,8 +154,12 @@ func (p *prod) ACRName() string {
 	return p.acrName
```

Comment:
I am thinking it's worth at least trying to change the acrName field to an acrDomain field to minimise confusion/duplication.